chatwoot/chatwoot
# feat: Adds the ability to send messages by clicking `Cmd+Enter` key

# Pull Request Template

## Description

Now we have only able to send messages by clicking Enter key or send button. This PR adds the ability to send messages by clicking Command plus Enter key only if you enable it in profile settings. 

Fixes #3972 

## Type of change


## How Has This Been Tested?

1. Open profile settings.
2. Now you can see the settings for enabling `Send message`.

<img width="1357" alt="Screenshot 2022-05-09 at 1 33 44 PM" src="https://user-images.githubusercontent.com/64252451/167367652-6f4646dc-8c44-4dd5-99b1-3e7beacb6f8e.png">

3. Select the card which you want and go to conversation view.
4. Now you can see the `Send (↵)` changes to  `Send (⌘ + ↵)` in both editors.
     
**cases**

1. If we turn on Cmd+Enter to send a message
https://www.loom.com/share/b06852ba798542c787809d2448d7cdb1

2. If we turn off Cmd+Enter to send a message
https://www.loom.com/share/cc2b0fe353584c86b29cc71a1529b7d0


## Checklist:
